docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -494,3 +494,16 @@ More information on this topic can be found on the link:cdi-reference#bean_disco
 == Building with `./gradlew build`
 
 Starting from 1.1.0.Final, `./gradlew build` will no longer build the native image. Add `-Dquarkus.package.type=native` build argument explicitly as explained above if needed.
+
+== Publishing your application
```

Comment:
@aloubyansky I added this piece of docs to explain why we have the `enforcedPlatform` and how to publish the application with it. WDYT ?

Reply:
Thanks, looks good.